tests/test_login.py

```diff
@@ -68,3 +68,35 @@ def test_get_session_user(self, client):
         retrieved_user_id = client.get_session_user(session_id=session_id)
 
         assert retrieved_user_id == user_id
+
+    def test_login_wrong_user(self, client):
+        try:
+            user_id, session_id = client.login('wrong-user!', '')
+            assert False
+        except hydra_base.exceptions.HydraLoginUserNotFound as e:
+            assert True
+        except:
+            assert False
+
+
+    def test_login_wrong_password(self, client):
+        try:
```

Comment:
as above, please use the standard way of testing for exceptions

Reply:
done it